### PR TITLE
000 underline bug

### PR DESCRIPTION
**Pull request by JakubRS7**

### CODE REVIEW

**Review by lexseaton — APPROVED** [review]

**Review by hannahmorgan24 — APPROVED** [review]

File: views/openJobRoleList.html [hunk 1]
```diff
@@ -4,26 +4,26 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     {% include "links.html" %}
-    <title>Open Job Role List</title>
+    <title id="">Open Job Role List</title>
```
Comment (boydstepkainos) [inline views/openJobRoleList.html]: Why empty?

Comment (lexseaton) [reply views/openJobRoleList.html]: i dont think title needs an id as it's the tab of the page so wouldn't be used

Comment (boydstepkainos) [reply views/openJobRoleList.html]: then remove the id element? better than having a blank one

Comment (boydstepkainos) [reply views/openJobRoleList.html]: can someone update the ticket to document that as an aside, the id's were added under it

**Review by boydstepkainos — APPROVED** [review]

**Pull request merged**